src/services/selector.js

```diff
@@ -1,6 +1,22 @@
 import { finder } from '@medv/finder/finder.js'
 
-export default function selector(e, { dataAttribute } = {}) {
+export default function selector(e, { dataAttribute, resolverAttribute, exclude } = {}) {
+  let parent = e.target
+  while (parent && parent.id != exclude) {
+    parent = parent.parentNode
+  }
+  if (parent && parent.id == exclude) {
+    return false
```

Comment:
I think returning empty string makes more sense with this function (and avoid mixing return types)